src/sentry/api/endpoints/organization_teams.py

```diff
@@ -81,6 +81,7 @@ def get(self, request, organization):
 
         :pparam string organization_slug: the slug of the organization for
                                           which the teams should be listed.
+        :param string detailed: Specify "0" to return team details that do not include projects
```

Comment:
Would a better name be `sparse`? And then it can do the inverse here.

Reply:
We use `detailed` elsewhere, and I think that makes sense in terms of how we'd like our API to behave. Ideally our default behaviour becomes not detailed, and people have to explicitly ask for detailed, and so pass `detailed=1`

Reply:
See this thread https://github.com/getsentry/sentry/pull/13925#discussion_r301767480